src/main/java/frc/robot/Constants.java

```diff
@@ -19,4 +19,22 @@ public final class Constants
         public static final int NUM_DIGITS_IN_TIME         = 8;
         public static final int HEADING_WIDTH              = 80;
     }
+
+    public static class Manipulator
+    {
+        public static final int WRIST_MOTOR_CAN_ID         =      0;
+        public static final int WRIST_ENCODER_PORT         =      0;
+        public static final double WRIST_MIN_ANGLE         =  -90.0;
+        public static final double WRIST_MAX_ANGLE         =   90.0;
+        public static final int TWIST_MOTOR_CAN_ID         =      0;
+        public static final int TWIST_ENCODER_PORT         =      0;
+        public static final double TWIST_MIN_ROTATION      = -180.0;
+        public static final double TWIST_MAX_ROTATION      =  180.0;
+        public static final int GRASP_MOTOR_CAN_ID         =      0;
+        public static final int GRASP_ENCODER_PORT         =      0;
+        public static final double GRASP_MIN_POSITION      =    0.0;
+        public static final double GRASP_MAX_POSITION      =    0.0;
+        public static final int INTAKE_MOTOR_CAN_ID        =      0;
+        public static final int INTAKE_SENSOR_PORT         =      0;
```

Comment:
Good job aligning the `=` signs, but don't align the numbers after them. Unless the decimal points are aligned, it can be super tricky to keep track of what all the numbers are if you're just skimming. Aligning the decimal points looks super bad though, so don't do that.